packages/gatsby-plugin-page-creator/src/create-pages-from-collection-builder.ts

```diff
@@ -11,13 +12,13 @@ import { collectionExtractQueryString } from "./collection-extract-query-string"
 import { isValidCollectionPathImplementation } from "./is-valid-collection-path-implementation"
 import { CODES, prefixId } from "./error-utils"
 
-// TODO: Do we need the ignore argument?
```

Comment:
No, because before this function is called, ignore is already handled: 

https://github.com/gatsbyjs/gatsby/blob/fs/pass-slugify-options/packages/gatsby-plugin-page-creator/src/create-page-wrapper.ts#L38-L41